frontend_tests/node_tests/compose.js

```diff
@@ -256,6 +257,76 @@ people.add(bob);
     assert($("#compose-all-everyone").visible());
 }());
 
+(function teste_markdown_shortcuts() {
+    blueslip.error = noop;
+    blueslip.log = noop;
+
+    var queryCommandEnabled = true;
+    global.document.queryCommandEnabled = function () {
+        return queryCommandEnabled;
+    };
+
+    var event = {
+        keyCode: 66,
+        target: {
+            id: 'compose-textarea',
+        },
+        stopPropagation: noop,
+        preventDefault: noop,
+    };
+    var input_text = "";
+    var range_start = 0;
+    var range_length = 0;
+
+    $("#compose-textarea").range = function () {
+        return {
+            start: range_start,
+            end: range_start + range_length,
+            length: range_length,
+            range: noop,
+        };
+    };
+    $('#compose-textarea').caret = noop;
+
+    // Test bold- ctrl/cmd + b.
+    input_text = "anything bold";
+    $("#compose-textarea").val(input_text);
+    event.keyCode = 66;
+    event.metaKey = false;
+    event.ctrlKey = true;
+    range_length = input_text.length;
+    compose.handle_keydown(event);
+    // Test if no text is selected.
+    range_length = 0;
+    compose.handle_keydown(event);
```

Comment:
The test setup here is good, but we need to make assertions about what actually happens here, such as does the text get updated?